README.md

```diff
@@ -511,6 +544,54 @@ ensures that as new modalities are added, and current modalities are extended, n
+
+### The CEP-78 Page System
+
+Ownership of NFTs within a CEP-78 contract is now tracked with a series of `pages`, with each page tracking a range of 1,000 tokens each. When installing an instance of the CEP-78 contract, the user determines the total token supply. This, in turn, determines the maximum number of pages, i.e., for a 10,000 token collection, each account could have up to 10 pages numbering from 0-9 tracking ownership of NFTs.
+
+A `page_table` tracks which pages within a range have been allocated and set for a certain user. The size of the page table directly correlates to the total token supply, i.e. for a CEP-78 instance tracking 10,000 tokens, the page table would be 10 bits wide. For a total of 20,000 it would be 20 bits wide. The cost of the initial page table allocation depends on the overall total size of a collection, with larger collections possessing correspondingly greater gas costs. To make initial minting costs more stable across contracts, the process of allocating a page table has been shifted to the `register_owner` entry point.
+
+After registering as an owner, the contract creates an entry within the `page_table` dictionary for the minting account or contract. This dictionary entry consists of a series of `boolean` values amounting to the total number of pages in the collection. In our 10,000 token example, this would be 10 bits set to false.
+
+Upon minting the token, the user will pay for a page allocation. This adds them to the `page` dictionary, in which each entry corresponds to a specific account or contract that owns tokens within that page. That account or contract's entry in the `page` dictionary will consist of 1,000 `page_address` bits set to `False` upon allocation, and the minting of any given token in that page will set the `page_address` bit to `True`.
+
+In addition, that account or contract's `page_table` will be updated by marking the corresponding page number's bit as `True`.
+
+As an example, consider a new user minting their first NFT with a given CEP-78 contract set to a maximum number of 10,000 tokens. They are minting the 2,350th token within that collection. The following sequence of events would occur:
+
+1) The contract registers their account as an owner.
+
+2) The contract creates a `page_table` dictionary for that account, with 10 boolean values. As the numbering system begins with `0`, the third boolean value corresponding with page `2` is set to `True`.
+
+3) The account pays for allocation of page 2, creating an entry in the `Page 2` dictionary for that account. Within that entry, there are 1,000 boolean values set to false. Minting the 2,350th token in the collection sets the corresponding `page_address` boolean for 350 as `True`.
```

Comment:
Wouldn't the 2,350th token be at position 349 instead of 350 on the page?
In the next paragraph, we have the 3,000th token in position 2,999.
I have a similar question for the 5,125th token. Would the page_address boolean for 125 be an index=124? I am not really understanding just by looking at the readme.

Reply:
I think you're correct here, I'll get a quick PR up to address this. Some of your other comments were addressed in the follow-up PR to this https://github.com/casper-ecosystem/cep-78-enhanced-nft/pull/108